README.md

```diff
@@ -35,7 +36,11 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+app.put("#", (item, res) => {
+  Instructor.findOneAndUpdate({name: "Andy"}, item.body, {new: true}, (docs) => {
+    console.log(docs);
+  })
+})
 ```
```

Comment:
What exactly is in `item.body`? Sounds like you might be replacing an instructor with a wishlist item.